docs/features/series.md

```diff
@@ -119,23 +116,23 @@ GET playlist\<playlistid>
      "medaid":"dwEE1oBP",
      "title":"Video Title",
      "description":"Lorem ipsum",
-     "seriesId": "aSZZ1oBP",
+     "contentType": "series",
      "images":[],
      "sources":[],
      "tracks":[]
  }
 ]
 ```
 
-We do not show episode number and season number for separate series episodes. If you click on such an item, you will see a simple movie page.
+We can also show episode number and season number for separate series episodes in case they have `seasonNumber` and `episodeNumber` custom params.
```

Comment:
Doesn't this data come from the series metadata, not custom params?

Reply:
I'm not sure this is something we want to encourage. It's a lot of overhead and can easily be set wrong where the custom params wouldn't match the episodes placement in an actual series. I think it would be better long term to just use the episode content type and add an endpoint to the series service to get the series and episode number of any media item (though this will run into the issue that we created when we allowed an item to be in more than 1 series.)

Reply:
I agree that the use case is not clear here. So we should not encourage it. However we can also find the parent id by an episode item in case it has an `episode` content type. I will remove this paragraph for now. Probably we can come back to discussing it when starting work on Favorites and Continue Watching section.

Cases I am thinking about:

1. Series in Favorites. We decided to mark the whole Series object as a favorite one.

2. Series in Continue Watching. It looks like here we need an episode and not a whole object. We already try to get a parent media item and show series page for `episode` items. So it should work out of the box in case there is a content type set. But what tag should we use for such items in Continue Watching shelf? What if we show something like an `episode` tag instead of season / episode number for separate episodes as part of Continue Watching section? We already show `series` tag so it makes sense.